server/src/data/patchApp.js

```diff
@@ -0,0 +1,43 @@
+const joi = require('joi')
+
+const paramsSchema = joi
```

Comment:
the existing updateApp did too many things and I didn't want to use it - although this in theory can be used to update any properties in the app object, for now, I am restricting, through the joi validation, it to only the d2config and changelog.